skeleton/Rakefile

```diff
@@ -1,5 +1,6 @@
 require 'puppetlabs_spec_helper/rake_tasks'
-require 'puppet/vendor/semantic/lib/semantic'
+require 'puppet/version'
+require 'puppet/vendor/semantic/lib/semantic' unless Puppet.version.to_f < 3.6
```

Comment:
I'm not sure I follow. This is including semantic unless the Puppet version is too low to have semantic included in it.